addon/data/set-installed-flag.js

```diff
@@ -4,9 +4,8 @@
  * http://mozilla.org/MPL/2.0/.
  */
 
-/* global cloneInto unsafeWindow */
+/* global unsafeWindow self */
 
 // Allow friendly clients to know that Test Pilot is installed.
-
 unsafeWindow.navigator.testpilotAddon = true;
```

Comment:
this one is for the non-testpilot sites, i.e. the `whitelist` in `addon/src/lib/environments.js`